mrp_bom_version/views/res_config_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="mrp_config_settings_versioning">
+            <field name="name">mrp.config.settings.versioning</field>
+            <field name="model">mrp.config.settings</field>
+            <field name="inherit_id" ref="mrp.view_mrp_config" />
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='module_mrp_operations']" position="after">
+                    <field name="group_mrp_bom_version" widget="upgrade_boolean"/>
```

Comment:
You shouldn't use this widget, as it uses only for enterprise options. You should use `toggle_button` or `boolean_option`